rules/gb-model.smk

```diff
@@ -602,13 +602,24 @@ rule compose_network:
     input:
         unpack(input_profile_tech),
         unpack(demands),
+        **{
+            f"regional_{flexibility_type}": resources(
+                f"gb-model/regional_{flexibility_type}.csv"
+            )
+            for flexibility_type in config["fes"]["gb"]["flexibility"][
+                "Technology Detail"
+            ].keys()
+        },
         network=resources("networks/base_s_{clusters}.nc"),
         powerplants=resources("gb-model/fes_powerplants.csv"),
         tech_costs=lambda w: resources(
             f"costs_{config_provider('costs', 'year')(w)}.csv"
         ),
         hydro_capacities=ancient("data/hydro_capacities.csv"),
         chp_p_min_pu=resources("gb-model/chp_p_min_pu_{clusters}.csv"),
+        ev_demand_shape=resources("gb-model/ev_demand_shape_s_{clusters}.csv"),
+        ev_demand_peak=resources("gb-model/regional_fes_ev_unmanaged_charging.csv"),
+        ev_storage_capacity=resources("gb-model/regional_fes_ev_storage.csv"),
```

Comment:
Need to be careful here that we're not double counting. EV/transport demand files will also be picked up in the `unpack(demands)` call above, and then processed inside `compose_network`. Is it adding transport demand twice at the moment...?